### PR TITLE
feat: card component

### PR DESCRIPTION
**Pull request by Averethel**
References [link the ticket here]

## Motivation and context

When implementing [product cards](https://www.figma.com/file/dMPSVZaUu1WZOmfVtHnqhO/Insertion-PART-2?node-id=142%3A43459&t=Q5JLFOEsjK6NzLNf-0) it turned out we don't export `Card` component and its parts from chakra.

This PR exports them and adds a very basic style

## How to test

Interact with `Card` in storybook

### CODE REVIEW

File: src/themes/components/card.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  header: {
+    paddingX: '2xl',
+    marginY: '2xl',
+  },
+  body: {
+    paddingX: '2xl',
+    marginY: '2xl',
+  },
+  footer: {
+    borderTop: '1px solid',
```
Comment (GutierrezGo) [inline src/themes/components/card.ts:23]: I think you can go only with `1px` from our tokens

```suggestion
    borderTop: '1px',
```

**Review by GutierrezGo — COMMENTED** [review]
One small question:
Should the Footer content  be aligned with the title and body contents?

Screenshot 2023-03-09 at 09 37 53

**Comment by Averethel** [comment]
@GutierrezGo it doesn't seem so. From the card design, they have different padding and it is intentional. 
I'll verify with Sergiu when he's back from his holidays and adapt if needed

File: src/components/card/index.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  maxWidth,
+}) => (
+  <Card maxW={`${maxWidth}px`}>
+    {tileContent ? <CardHeader>{tileContent}</CardHeader> : null}
+    {bodyContent ? <CardBody>{bodyContent}</CardBody> : null}
+    {footerContent ? <CardFooter>{footerContent}</CardFooter> : null}
+  </Card>
+);
+
+# Top Listing Badge
```
Comment (Averethel) [inline src/components/card/index.stories.mdx:37]:
```suggestion
# Card
```

**Review by Milos-Zbiljic — APPROVED** [review]

**Pull request merged**